plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/product-details/style.scss

```diff
@@ -18,7 +19,7 @@
 	display: inline-block;
 }
 
-.is-large:not(.wc-block-checkout) {
```

Comment:
Good idea to test subscriptions. The checkout and cart styles have departed a lot since then so I didn't think too much of it not being needed anymore. 

Subscriptions is an extension though right, shouldn't any CSS fix for that specific case belong in that plugin?